### ISSUE TITLE
java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter

### ISSUE THREAD

**Issue by michaellindon** [metasoarous/oz#20]
I'm getting the following error

```
CompilerException java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter, compiling:(aleph/http/client_middleware.clj:420:7)
```
:(

**Comment by realgenekim** [comment]
@michaellindon Just add this to your `project.clj` file — I suspect this is because you're using Java 9+?

```
  ; to prevent error on loading oz in repl
  :jvm-opts ~(let [version     (System/getProperty "java.version")
                   [major _ _] (clojure.string/split version #"\.")]
               (if (>= (java.lang.Integer/parseInt major) 9)
                 ["--add-modules" "java.xml.bind"]
                 []))
```

(I've seen this posted a couple of places, including here.  https://github.com/bhauman/lein-figwheel/issues/612)

**Comment by metasoarous** [comment]
@realgenekim Thanks for chiming in!

It looks like there was an [aleph update which also resolves this issue](https://github.com/ztellman/aleph/issues/383), so I'm going to try to update that lib in my next release to resolve this problem. In the mean time, if you try specifying the latest aleph lib above oz in your dependency tree, this should also resolve the problem.

@michaellindon Please let me know if you're able to get either of these solutions to work, and thanks for your patience here.

**Comment by metasoarous** [comment]
@michaellindon @realgenekim I just released `[metasoarous/oz "1.4.0-SNAPSHOT"]`, if one of you would be so kind as to test that out with Java > 8, and see if it removes the need for those fancy `:jvm-opts`.

Thanks in advance!

**Comment by metasoarous** [comment]
I just released `1.4.0`, and this issue should be fixed there. Please reopen if you still have this problem after switching.
